js/build.js

```diff
@@ -7,23 +7,4 @@ $('[data-dynamic-lists-id]').each(function(){
   if (!data.layout) {
     dynamicLists[id] = new DynamicLists(data, container);
   }
-
-  if (data.layout === 'simple-list' || data.layout === 'news-feed') {
```

Comment:
Your previous change showed that line clamping applies to the news feed layout as well.

But, I don't see any changes made to the news feed template in this PR.

Reply:
@tonytlwu Yes, that is my mistake I'll make another commit to this PR to fix that.